src/docs/background/pricing/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Pricing
+slug: pricing
+updated: 2020-06-17
+category: background
+ingress:
+  Flex allows lots of flexibility for your providers in terms of how
+  they can set their pricing. This guide walks you through how price is
+  defined in Flex.
+published: true
+---
+
+_**NOTE! Current default pricing method has deprecated our old custom
+pricing feature. If you are looking for the old custom pricing
+background article, it can be found
+[here](https://5ee94c280d38f10008a3bfa1--sharetribe-flex-docs-site.netlify.app/docs/background/custom-pricing/).**_
```

Comment:
Is this link going to be left as is ponting to the netlify build?

Reply:
@sktoiva yes, I think that is the way we've referred to old articles/versions.